Client/TabManagement/TabManagerImplementation.swift

```diff
@@ -215,15 +217,8 @@ class TabManagerImplementation: LegacyTabManager, Notifiable {
     // MARK: - Save tabs
 
     override func preserveTabs() {
-        // If there are no tabs then we do not need to preserve as we are initalizing the app
-        // (there should always be at least a homepage)
-        // Also if the a restore is in progress then presevering tabs could cause issues
-        if tabs.isEmpty || isRestoringTabs {
-            logger.log("Attempted to preserve tabs while tabs were restoring or empty, tab count \(tabs.count), isRestoring \(isRestoringTabs)",
-                       level: .fatal,
-                       category: .tabs)
```

Comment:
Removed this log because we now know the scenarios this happens and is no longer useful, especially not as a fatal.